privacyidea/webui/login.py

```diff
@@ -58,7 +58,8 @@ def single_page_application():
 
     if current_app.config.get("PI_UI_DEACTIVATED"):
         # Do not provide the UI
-        return render_template("deactivated.html")
+        return current_app.response_class(render_template("deactivated.html"),
```

Comment:
We often use the ``current_app.respsonse_class(render_template...)`` which return an html mimetype. Would it be nice to have a wrapper funktion for this? 
(e.g. the same in webui/certificate.py)

Reply:
Fixed in 12f8421e